python/lib/client/setup.py

```diff
@@ -0,0 +1,24 @@
+from setuptools import setup, find_namespace_packages
+
+try:
+    with open('README.md', 'r') as readme:
+        long_description = readme.read()
+except:
+    long_description = ''
+
+exec(open('dmod/client/_version.py').read())
+
+setup(
+    name='dmod-client',
+    version=__version__,
+    description='Client interface package for components of the National Water Model as a Service architecture',
```

Comment:
"the National Water Model as a Service architecture" -> "the DMOD architecture"?

Reply:
Good catch.  I'll fix.